### ISSUE TITLE
Fix usage of project specific tools like Drush and Drupal Console

### ISSUE THREAD

**Issue by aleksijohansson** [wunderkraut/wundertools-image-fuzzy-developershell#36]
https://github.com/wunderkraut/wundertools-dockerprototype default layout has changed, so currently we are including the project specific binaries for Drush and Drupal Console from the wrong folder. Also the aliases for those are not working and some Drupal Console configuration is missing.

**Comment by james-nesbitt** [comment]
it's a good assumption that the project will be mounted at ${HOME}/project, and anyone not doing that, can assume responsibility for overriding the .zshrc

**Comment by james-nesbitt** [comment]
should we go for a ${HOME}/zshrc.d folder , and include everything in there?  then any impementation could add their own start up vars and not have to worry about overriding the whole .zshrc file.

**Comment by aleksijohansson** [comment]
@james-nesbitt That's actually a solid idea! I'll keep this issue open and cook up another PR for that.

**Comment by aleksijohansson** [comment]
@james-nesbitt Should we include our default configuration also in the zshrc.d folder (and have the .zshrc file only include files from the zshrc.d folder and do nothing else) so that one can replace the default configuration completely (by replacing the default config file in zshrc.d) without losing the ability to include other configuration in other files in zshrc.d?

**Comment by james-nesbitt** [comment]
agreed.  it's not very standard for zsh usage, so it won't be typically overriden this way, but it is much more extensible.

What I mean is that most users aren't using a .zshrc.d folder anywhere else, so they won't have a simple folder to overload.  Typically people can use this image as a base image, and add that plugins/settings that they want.

For the dockerprototype, we may even consider adding a .zshrc.d folder to demo/init projects, just as we do for drush and drupal-console.

**Comment by james-nesbitt** [comment]
#41 provides a first zshrc split.  The original .zshrc has been moved into .zshrc.d, and the new .zshrc file sources all contents of that directory (alphabetic order)

**Comment by aleksijohansson** [comment]
I think now after #41 we can see this as fixed.
